jobbergate-cli/Makefile

```diff
@@ -0,0 +1,52 @@
+# SETTINGS
+# Use one shell for all commands in a target recipe
+.ONESHELL:
+.DEFAULT_GOAL:=help
+SHELL:=/bin/bash
+PACKAGE_NAME:=jobbergate_cli
+
+.PHONY: install
+install:
+	poetry install
+
+.PHONY: test
+test: install
+	poetry run pytest
+
+.PHONY: lint
+lint: install
+	poetry run black --check ${PACKAGE_NAME}
+	poetry run isort --check ${PACKAGE_NAME}
+	poetry run flake8 --max-line-length=120 --max-complexity=40 ${PACKAGE_NAME}
```

Comment:
what about having a .flake8 file here? Example: https://github.com/omnivector-solutions/armada-api/blob/main/.flake8

Reply:
I prefer not to have an extra configuration file. I wish flake8 supported pyproject.toml like the other tools we use, but it won't (https://github.com/PyCQA/flake8/issues/234#issuecomment-812800832). Instead of adding another config file just for flake8, I prefer to specify the config in the Makefile.